build.sbt

```diff
@@ -23,13 +23,15 @@ developers := List(
   )
 )
 
+scalaVersion := "2.13.1"
 crossScalaVersions := List("2.10.7", "2.11.12", "2.12.10", "2.13.1")
 
 libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "3.1.0",
-  "org.seleniumhq.selenium" % "selenium-java" % "2.45.0", 
-  "org.eclipse.jetty" % "jetty-server" % "9.4.12.v20180830" % "test",
-  "org.eclipse.jetty" % "jetty-webapp" % "9.4.12.v20180830" % "test"
+  "org.seleniumhq.selenium" % "selenium-java" % "3.141.59",
+  "org.seleniumhq.selenium" % "htmlunit-driver" % "2.36.0",
```

Comment:
htmlunit-driver is not a transitive dependency anymore, so it needs to be added explicitly. This is the version compatible with Selenium 3.141.59.